schemas/rooms.delete.json

```diff
@@ -0,0 +1,11 @@
+{
+  "id": "rooms.delete",
+  "additionalProperties": false,
+  "type": "object",
+  "properties": {
+    "id": {
+      "type": "string",
+      "pattern": "[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}"
```

Comment:
А там вроде не было такого в стандарте. Если есть - то можно поменять

> On 08 Jul 2016, at 15:39, Aleksandr Sorokin <notifications@github.com> wrote:
> 
> In schemas/rooms.delete.json:
> 
> > @@ -0,0 +1,11 @@
> > +{
> > +  "id": "rooms.delete",
> > +  "additionalProperties": false,
> > +  "type": "object",
> > +  "properties": {
> > +    "id": {
> > +      "type": "string",
> > +      "pattern": "[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}"
> "format": "uuid"
> 
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub, or mute the thread.
>